monkey/monkey_island/cc/ui/src/components/pages/ConfigurePage.js

```diff
@@ -38,6 +38,7 @@ const CONFIGURED_PROPAGATION_CREDENTIALS_URL = '/api/propagation-credentials/con
 const configSubmitAction = 'config-submit';
 const configExportAction = 'config-export';
 const configSaveAction = 'config-saved';
+const CREDENTIALS_COLLECTORS_CONFIG_PATH = 'credentials_collectors';
```

Comment:
This is related to the above comment. Move this constant to the relevant file and import it like we've done for exploiters.